extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/LuceneFieldConfig.java

```diff
@@ -104,6 +110,11 @@
         if (StringUtils.isNotEmpty(cond)) {
             this.condition = Optional.of(cond);
         }
+
+        final String binaryStr = configElement.getAttribute(ATTR_BINARY);
+        if (StringUtils.isNotEmpty(binaryStr)) {
```

Comment:
would [isBlank](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-) a better choice?

Reply:
I am pretty sure it extends isNotEMpty() with a whitespace check?